src/wallet/walletActions.js

```diff
@@ -1,7 +1,50 @@
+import steem from 'steem';
 import { createAction } from 'redux-actions';
+import { createAsyncActionType } from '../helpers/stateHelpers';
+import { getTransactionHistory, getDynamicGlobalProperties } from '../helpers/apiHelpers';
 
 export const OPEN_TRANSFER = '@wallet/OPEN_TRANSFER';
 export const CLOSE_TRANSFER = '@wallet/CLOSE_TRANSFER';
+export const GET_GLOBAL_PROPERTIES = createAsyncActionType('@wallet/GET_GLOBAL_PROPERTIES');
+export const GET_USER_TRANSACTIONS = createAsyncActionType('@users/GET_USER_TRANSACTIONS');
+export const GET_USER_EST_ACCOUNT_VALUE = createAsyncActionType(
+  '@users/GET_USER_EST_ACCOUNT_VALUE',
+);
 
 export const openTransfer = createAction(OPEN_TRANSFER);
 export const closeTransfer = createAction(CLOSE_TRANSFER);
+
+export const getGlobalProperties = () => (dispatch) => {
+  dispatch({
+    type: GET_GLOBAL_PROPERTIES.ACTION,
+    payload: {
+      promise: getDynamicGlobalProperties().then(result => ({
+        ...result,
```

Comment:
What does it do? Won't it result in exactly the same object?

Reply:
yeah you're right lol, I'm just to use copying objects, will change though